ckanext/datastore/blueprint.py

```diff
@@ -36,17 +34,28 @@
 default = cast(ValidatorFactory, get_validator(u'default'))
 unicode_only = get_validator(u'unicode_only')
 
-DUMP_FORMATS = u'csv', u'tsv', u'json', u'xml'
 PAGINATE_BY = 32000
 
 datastore = Blueprint(u'datastore', __name__)
 
 
+def dump_formats():
+    dump_formats = [
+        formats.CSV(),
+        formats.TSV(),
+        formats.JSON(),
+        formats.XML(),
+    ]
+    for plugin in p.PluginImplementations(IDatastoreDump):
+        dump_formats.append(plugin())
```

Comment:
might want to `extend` here so that a plugin can define more than one format or pass in the current list and have the plugin return a modified list so that plugins could override/remove other formats (like removing the CSV/TSV ones and replacing them with ones that set `bom=True`)